Dockerfile

```diff
@@ -0,0 +1,18 @@
+### BUILD FRONT
+# invoices
+FROM node:9-alpine AS invoices-builder
+WORKDIR /usr/src/app
+COPY ui/ui-invoices/package.json ui/ui-invoices/yarn.lock ./
+RUN yarn install
+COPY ui/ui-invoices/ .
+RUN yarn build
+
+FROM node:9-alpine AS kmille-invoices
+WORKDIR /usr/src/app
+COPY package.json lerna.json ./
+COPY packages ./packages
+COPY api/api-invoices ./api/api-invoices
+RUN yarn install
+COPY --from=invoices-builder /usr/src/app/build ./api/api-invoices/build
+RUN node api/api-invoices/src/index.js
```

Comment:
This is not `RUN` this is `ENTRYPOINT`

- `RUN` is a buildtime command
- `ENTRYPOINT` is a runtime command